build/xwalk.gni

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) 2016 Intel Corporation. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# This file contains Crosswalk-feature-related build flags. Users should
+# import it into their build folder by run:
+#   echo 'import("//xwalk/build/xwalk.gni")' > out/your_folder/args.gn'
+# Then run:
+#   gn gen out/your_folder
+
+# Set it false when building content_shell.
+declare_args() {
+  is_build_crosswalk = true
+}
+
+if (is_build_crosswalk) {
+  target_os = "linux"
+  target_cpu = "x64"
+  is_debug = false
+
+  # From src/build/config/features.gni
+  # TODO: Enable nacl for Linux.
+  enable_nacl = false
+
+  # From src/build/config/features.gni
+  enable_plugins = true
+
+  # From src/build/config/features.gni
+  enable_webvr = false
+
+  # From src/third_party/widevine/cdm/widevine.gni
```

Comment:
There's no need to set this here since commit b5f0d1330f484e6c3b3c8ceeed5c94dd8689f684, the default is `false`. I'll remove it from the gyp version.